qa updatinge links on komm release notes, kba intro, storage csi (#3166)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5/storage/intro-csi/index.md

The CSI standard defines a set of services to reduce coupling with storage drives and provide more flexibility. These services are deployed along side a storage driver as sidecar containers. The sidecar containers are responsible for interacting with the Kubernetes API.

> [Konvoy will automatically configure native storage drivers][automated-storage] for the supported cloud provisioners (AWS, Azure, or GCP) which deploy the following sidecar containers:

Reviewer: ```suggestion
[Konvoy automatically configures native storage drivers][automated-storage] for the supported cloud provisioners (AWS, Azure, or GCP) that deploy the following sidecar containers:
```

In Konvoy, you can use CSI volumes through the Kubernetes storage API objects like PersistentVolumeClaims (PVCs), PersistentVolumes (PVs), and StorageClasses.

> Here's an example of a StorageClass for the [AWS EBS CSI driver][aws-ebc-csi-driver]. Save it in file named `ebs-sc.yaml`:

Reviewer: ```suggestion
Here is an example of a `StorageClass` for the [AWS EBS CSI driver][aws-ebc-csi-driver]. Save it in a file named `ebs-sc.yaml`:
```
